src/taskBuilder/views.py

```diff
@@ -340,7 +340,7 @@ def get(self):
 class CoursesTeachingView(MethodView):
 
     def get(self):
-        courses = current_user.get_courses_where_teacher_or_admin()
+        courses = current_user.get_courses_where_teacher_or_ta()
         courses = [c.serialize for c in courses]
         return flask.json.dumps(courses)
 
```

Comment:
before it was doing... 
        tas = current_user.get_courses_where_ta()
        courses = [c.serialize for c in current_user.coursesTeaching]
        courses += [c.serialize for c in tas]

I think we need to add in where current user is teaching as well right..